scripts/k6/event-apply-test .js

```
@@ -0,0 +1,53 @@
+    const url = `${BASE_URL}/api/events/${EVENT_ID}/register`;
+    const params = {
+        headers: {
+            'Content-Type': 'application/json',
+            Authorization: `Bearer ${token}`,
+        },
+    };
+
+    // 요청 보내기
+    let res = http.post(url, {}, params);
+
+    // 검증 & 실패 로그
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'res' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)